src/lib/SCEditor.js

```diff
@@ -355,7 +355,7 @@ define(function (require) {
 				}
 
 				if (options.autoExpand) {
-					base.expandToContent();
+					base.expandToContent(options.ignoreMaxHeight);
```

Comment:
Why didn't you change
https://github.com/samclarke/SCEditor/blob/master/src/lib/SCEditor.js#L491
too?